Tests/SwiftyJSON/NumberTests.swift

```diff
@@ -109,12 +109,12 @@ class NumberTests: XCTestCase {
         json.float = 23231.65
         XCTAssertTrue(json.float! == 23231.65)
         XCTAssertTrue(json.floatValue == 23231.65)
-        XCTAssertEqual(json.numberValue, NSNumber(value:23231.65))
+        XCTAssertEqualWithAccuracy(json.numberValue as Double, 23231.65, accuracy: 0.11)
```

Comment:
Why `0.11`?  Seems a very large difference... Are we sure we aren't papering over a problem here?